tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -365,37 +379,41 @@ Internal::FriendInfo TTreeProcessorMT::GetFriendInfo(TTree &tree)
 
 ////////////////////////////////////////////////////////////////////////////////
 /// Retrieve the name of the first TTree in the first input file, else throw.
```

Comment:
This needs to be something like "Retrieve the names of each TTree in each of the input files, else throw."

Reply:
fixed